AddonArmTemplate/FullAzureMonitorMetricsProfile.json

```diff
@@ -281,7 +282,8 @@
       "properties": {
         "description": "[concat(variables('kubernetesRecordingRuleGroupDescription'), variables('version'))]",
         "scopes": [
-          "[parameters('azureMonitorWorkspaceResourceId')]"
+          "[parameters('azureMonitorWorkspaceResourceId')]",
```

Comment:
Can you include teraform to this?

Reply:
Thanks! My bad, I missed it. I will add it.

Reply:
Added terraform